tests/drivers/spi/spi_controller_peripheral/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -53,6 +53,7 @@
 	dut_spi_dt: test-spi-dev@0 {
 		compatible = "vnd,spi-device";
 		reg = <0>;
+		spi-max-frequency = <DT_FREQ_M(8)>;
```

Comment:
Why?

Reply:
because:
`devicetree error: 'spi-max-frequency' is marked as required in 'properties:' in /home/adko/zephyrproject/zephyr/dts/bindings/test/vnd,spi-device.yaml, but does not appear in <Node /soc/spi@4002f000/test-spi-dev@0 in '/home/adko/zephyrproject/zephyr/misc/empty_file.c'>`